spec/jobs/solidus_subscriptions/process_installment_job_spec.rb

```diff
@@ -8,4 +8,16 @@
 
     expect(checkout).to have_received(:process)
   end
+
+  context 'when handling #perform errors' do
+    it 'swallows error on #perfom error' do
```

Comment:
Just a nitpick, but you have already specify that this is running in the context of a `#perform` error, no need to repeat it in the `it` blocks (same below).